<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电子相册</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6">
    <item>
     <widget class="QWidget" name="proTree" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>项目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ProTree" name="treeView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="picShow"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_music">
    <property name="title">
     <string>音乐(&amp;M)</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_music"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProTree</class>
   <extends>QTreeView</extends>
   <header>../../../code/album/protree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
